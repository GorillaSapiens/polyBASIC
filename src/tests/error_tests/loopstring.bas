for i := "here" to "there" step "lightly"
next i
