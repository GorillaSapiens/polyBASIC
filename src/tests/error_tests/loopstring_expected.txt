ERROR: @2:42, NEXT ON FOR LOOP WITH STRING VARIABLE
